cmd/swarmctl/service/flagparser/flags.go

```diff
@@ -51,6 +51,8 @@ func AddServiceFlags(flags *pflag.FlagSet) {
 
 	flags.String("log-driver", "", "specify a log driver")
 	flags.StringSlice("log-opt", nil, "log driver options, as key value pairs")
+
+	flags.Int64("pids-limit", -1, "set container pid limit")
```

Comment:
Using -1 to represent "unlimited" feels inconsistent with our other parameters. Is this the way it works in Docker Engine? Unless there's some precedent, I'd rather make 0 correspond to unlimited and avoid negative values entirely.